app/services/katello/repository_type_manager.rb

```diff
@@ -0,0 +1,59 @@
+module Katello
+  class RepositoryTypeManager
```

Comment:
No - because my point is a fundamental design point. The precedent is a loose one here and I could just as easily point to another "plugin" system in use (https://github.com/Katello/bastion/blob/master/lib/bastion.rb#L9). As I said, I won't hold things up, but I'd be remiss if I didn't state my strong opinion on the matter for your consideration.